### PR TITLE
Minor: port more create_drop table tests to sqllogictests

### PR DESCRIPTION
**Pull request by jiangzhx**
# Which issue does this PR close?

Closes #.

# Rationale for this change

# What changes are included in this PR?

# Are these changes tested?

# Are there any user-facing changes?

### CODE REVIEW

File: datafusion/core/tests/sqllogictests/test_files/ddl.slt [hunk 1]
```diff
@@ -448,6 +448,16 @@ CREATE TABLE table_without_values(field1 BIGINT NULL, field2 BIGINT NULL);
 statement ok
 CREATE TABLE IF NOT EXISTS table_without_values(field1 BIGINT, field2 BIGINT);
 
+statement ok
+CREATE EXTERNAL TABLE aggregate_simple STORED AS CSV WITH HEADER ROW LOCATION 'tests/data/aggregate_simple.csv'
+
+statement ok
+CREATE EXTERNAL TABLE IF NOT EXISTS aggregate_simple STORED AS CSV WITH HEADER ROW LOCATION 'tests/data/aggregate_simple.csv'
```
Comment (waynexia) [inline datafusion/core/tests/sqllogictests/test_files/ddl.slt:455]: According to the previous cases I think we need to create this table again without `IF NOT EXIST` and expect an error

File: datafusion/core/tests/sqllogictests/test_files/ddl.slt [hunk 2]
```diff
@@ -580,4 +601,4 @@ select * from t;
 0 years 0 mons 5 days 0 hours 0 mins 0.000000003 secs 1
 
 statement ok
-drop table t;
+drop table t;
```
[the remark this answers is not in the document]
Comment (jiangzhx) [reply datafusion/core/tests/sqllogictests/test_files/ddl.slt]: thanks

File: datafusion/core/tests/sqllogictests/test_files/ddl.slt (hunk 1 above)
Comment (jiangzhx) [reply datafusion/core/tests/sqllogictests/test_files/ddl.slt:459]: fixed,thanks again.
`CREATE EXTERNAL TABLE`  
need to check recreate the same EXTERNAL table

**Review by waynexia — APPROVED** [review]
Looking good, thanks 👍

**Comment by waynexia** [comment]
link to #4495

**Review by r4ntix — APPROVED** [review]

**Comment by alamb** [comment]
Thank you all!

**Pull request merged**